### ISSUE TITLE
Add OpenAlias Support 

### ISSUE THREAD

**Issue by anyone-ob** [bitcoin-wallet/bitcoin-wallet#533]
Hello,

please add support for OpenAlias.
OpenAlias is a way to map DNS / DNSSEC names to BTC addresses.

more details: https://openalias.org/

**Comment by lane-c-wagner** [comment]
Take a look at OpenCAP first. A better alias option IMO:
https://github.com/opencap/protocol

**Comment by Klexx** [comment]
@lane-c-wagner can you summarize why you think that ? I think the simpler the better in regards to adoption.

**Comment by lane-c-wagner** [comment]
Here is the differences I wrote up awhile ago: “Crypto alias comparison — OpenCAP vs OpenAlias” by Lane Wagner https://link.medium.com/gfezPgZi5X
To be honest though not much has happened with either protocol by way of adoption, so the devs should really look into this before implementing.

**Comment by schildbach** [comment]
I think neither of the proposals have gained traction. So I'm closing this.
